### ISSUE TITLE
There is an error in style code!

### ISSUE THREAD

**Issue by 9B8DY6** [xiaoqian-shen/MoStGAN#2]
![image](https://user-images.githubusercontent.com/67573223/234588088-badfddfc-f42c-4fbd-8954-f4df9c36786d.png)
variable style has 3 dimension ((16,8,518) shape) but your code consider it as 4 dim and trying .view to 'styles'.

**Comment by hellog2n** [comment]
![image](https://user-images.githubusercontent.com/63945045/235343652-aee984ff-1190-4faf-b4e9-9e35aecdd24e.png)

I have similar error too for using ffs pretrained network. Did you solve this problem?

**Comment by 9B8DY6** [comment]
@hellog2n 장유진 바보

**Comment by xiaoqian-shen** [comment]
Hi, thank you for bringing up this issue. I believe there is no problem with the code itself. It's possible that the provided pretrained weight of FFS has undergone some changes for analysis that are affecting the function input. I will update the FFS pretrained weight, or you can train it from scratch using the given code.

**Comment by ArefAz** [comment]
@9B8DY6 I encountered the same issue when trying to use `generate.py` to create sample videos, were you able to resolve it? I tried different pre-trained checkpoints but none of them worked.
